it/controllers/employment/EmploymentSummaryControllerISpec.scala

```diff
@@ -158,32 +157,32 @@ class EmploymentSummaryControllerISpec extends IntegrationTest with ViewHelpers
   }
 
   object CommonExpectedCY extends CommonExpectedResults {
-    val expectedH1: String = "PAYE employment"
+    val expectedH1: String = "Cyflogaeth TWE"
     val expectedTitle: String = expectedH1
 
-    def expectedCaption(taxYear: Int): String = s"PAYE employment for 6 April ${taxYear - 1} to 5 April $taxYear"
+    def expectedCaption(taxYear: Int): String = s"Cyflogaeth TWE ar gyfer 6 Ebrill ${taxYear - 1} i 5 Ebrill $taxYear"
 
     val notificationTitle: String = "Important"
-    val notificationContent: String = "You cannot have expenses without an employer. Add an employer or remove expenses."
-    val noEmployers: String = "No employers"
+    val notificationContent: String = "Ni allwch gael treuliau heb gyflogwr. Ychwanegu cyflogwr neu dynnu treuliau."
+    val noEmployers: String = "Dim cyflogwyr"
     val name: String = "maggie"
-    val startedDateString: LocalDate => String = (startedOn: LocalDate) => s"Started " + ViewUtils.translatedDateFormatter(startedOn)(getMessages(isWelsh = true))
-    val startedDateMissingString: String = "Start date missing"
-    val startedDateBeforeString: Int => String = (taxYear: Int) => s"Started before 6 April $taxYear"
+    val startedDateString: LocalDate => String = (startedOn: LocalDate) => s"Wedi dechrau ar " + ViewUtils.translatedDateFormatter(startedOn)(getMessages(isWelsh = true))
+    val startedDateMissingString: String = "Dyddiad dechrau ar goll"
+    val startedDateBeforeString: Int => String = (taxYear: Int) => s"Wedi dechrau cyn 6 Ebrill $taxYear"
     val add: String = "Add"
```

Comment:
There is a translation, but this variable is not used. I am deleting it.

Reply:
I think the content needs to be tested since it appears on the page. We can add the check in this ticket or I can add the checks when completing SASS-2410

Reply:
There are other bits that are not fully tested. Can you please add this as a test for SASS-2410? It is important that we merge this story as soon as possible.